warehouse/templates/index.html

```diff
@@ -0,0 +1,50 @@
+{% extends "base.html" %}
+
+{% block content %}
+<section class="index-page">
+    <header class="header">
+        <div class="flash-notice">
+            <p><strong class="site-name hidden-xs hidden-sm">Warehouse (pypi.python.org)</strong>
```

Comment:
This notice needs translated, also instead of hardcoding the name it should use ``request.registry.settings['site.name']``.